renderers/Compositor.js

```diff
@@ -243,6 +243,10 @@ Compositor.prototype.receiveCommands = function receiveCommands(commands) {
     for (var i = 0; i < len; i++) {
         this._inCommands.push(commands[i]);
     }
+    
+    for (var selector in this._contexts) {
```

Comment:
I agree, there is actually a TODO in the code, but I don't think this would belong in this PR. Should I change it?